earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -183,7 +189,7 @@ END
 
 final_assetroot_configuration()
 {
-    if [ "$IS_SLAVE" = true ]; then
+    if [ "$IS_SLAVE" == "true" ]; then
```

Comment:
Lint: `==` is not a POSX operator, '=' is.  In case we care about POSIX compliance.